sdk/communication/communication-common/src/identifierModelSerializer.ts

```diff
@@ -59,6 +63,14 @@ export interface SerializedPhoneNumberIdentifier {
    * The phone number in E.164 format.
    */
   value: string;
+  /**
+   * Value generated when the same number joins the same call multiple times.
```

Comment:
```suggestion
   * The asserted Id is set on a phone number that is already in the same call to distinguish from other connections made through the same number.
```
maybe we can use the same summary from .net, to provide more details